src/dynamics/mpas/dyn_comp.F90

```diff
@@ -32,15 +54,31 @@ module dyn_comp
 
     public :: dyn_debug_print
     public :: mpas_dynamical_core
+    public :: ncells, ncells_solve, nedges, nedges_solve, nvertices, nvertices_solve, nvertlevels
+    public :: ncells_global, nedges_global, nvertices_global, ncells_max, nedges_max
+    public :: sphere_radius
 
+    !> NOTE:
+    !> This derived type is not used by MPAS dynamical core. It exists only as a placeholder because CAM-SIMA requires it.
+    !> Developers/Maintainers/Users who wish to interact with MPAS dynamical core may do so by using the "instance/object"
+    !> below.
```

Comment:
A question about planned future development stemming this comment: do we not plan on using `dyn_import_t` and `dyn_export_t` ever when it comes to physics-dynamics coupling?

Reply:
From what I understand, the `dyn_import_t` and `dyn_export_t` derived types are used like a container of pointers to dynamical state variables.

This functionality is no longer needed because we now have a more generic facility: `get_variable_pointer` [[1]](https://github.com/ESCOMP/CAM-SIMA/blob/d18264ba2467694d053246789215e23c2048f176/src/dynamics/mpas/driver/dyn_mpas_subdriver.F90#L3139), which allows for drawing the reference of any internal variables from MPAS dynamical core.

Therefore, when implementing dynamics-physics coupling, we can just draw pointers to those necessary internal variables from MPAS dynamical core by calling `get_variable_pointer`.

Reply:
If you would like to preview how it works, you can look at the `implement-dyn-run` [[2]](https://github.com/kuanchihwang/CAM-SIMA/blob/develop/implement-dyn-run/src/dynamics/mpas/dyn_coupling.F90) branch of my fork, where I am currently implementing the time integration + dynamics-physics coupling of MPAS dynamical core in CAM-SIMA.

Reply:
Seems reasonable enough to me!